includes/definitions/discovery/witchos.yaml

```diff
@@ -0,0 +1,13 @@
+mib: TELTONIKA-MIB
+modules:
+    os:
+        serial: TELTONIKA-MIB::serial.0
+        hardware: TELTONIKA-MIB::productCode.0
+        version: TELTONIKA-MIB::fwVersion.0
+    processors:
+        data:
+            -
+                oid: TELTONIKA-MIB::cpuUsage
+                num_oid: '.1.3.6.1.4.1.48690.1.8.{{ $index }}'
+                index: cpuUsage.{{ $index }}
+                type: witchos-cpu
```

Comment:
Probably want to leave the type to the default (it will be witchos)

Reply:
Done